test/fixtures/out/converts-to-shorthand.astro

```diff
@@ -2,5 +2,4 @@
 const style = "color:red;";
 let id = "#aloha";
 ---
-
```

Comment:
These extra linebreaks were removed from our printer adding linebreaks between “documents” (with the code fence & HTML being treated as 2 documents).

If we want a line break back here, I think we should add it as the first child of the HTML document only in that instance; otherwise unconditionally adding linebreaks between documents sometimes results in too much whitespace.

Reply:
I can give `softline` a try